source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,294 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to run a :ref:`text search <text-search-go>`.
```

Comment:
Optional: Consider adding a clarifying note that MongoDB text search is different than [Atlas Search](https://docs.atlas.mongodb.com/atlas-search/), which has significantly different behavior and expanded functionality.